docs/guides/how_to_guides/creating_batches/how_to_load_a_pandas_dataframe_as_a_batch.rst

```diff
@@ -1,18 +1,145 @@
 .. _how_to_guides__creating_batches__how_to_load_a_pandas_dataframe_as_a_batch:
 
-How to load a Pandas dataframe as a Batch
+How to load a Pandas DataFrame as a Batch
 =========================================
 
-.. admonition:: Admonition from Mr. Dickens
+This guide will help you load a Pandas DataFrame as a Batch for use in creating expectations.
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+.. content-tabs::
 
+    .. tab-container:: tab0
+        :title: Show Docs for Stable API (up to 0.12.x)
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+        .. admonition:: Prerequisites: This how-to guide assumes you have already:
 
-If it would be useful to you, please comment with a +1 and feel free to add any suggestions or questions below.
+            - :ref:`Set up a working deployment of Great Expectations <tutorials__getting_started>`
+            - Configured a :ref:`Pandas/filesystem Datasource <how_to_guides__configuring_datasources>`
+            - Identified a pandas DataFrame that you would like to use as the data to validate.
+
+        Steps
+        -----
+
```

Comment:
I love that we have the prerequisites, but I wonder if we should start at the top of the notebook here (so to speak), starting with imports and instantiating a context, since we need this for the initial steps.

Reply:
I have gone back and forth on this, and I think it is a good suggestion but my hesitation is that we have several different ways to create / instantiate a data context. We have the CLI, in-code creation and finally the new `test_yaml_config` workflow. Rather than have all of those guides here in this doc, where they can get outdated - I was thinking "DRY" and reference the "Configuring Data Contexts" instead via the `Set up a working deployment of Great Expectations` link. I think you are right that there should be a more clear callout though - let me know what you think of the text in this commit: e3fc837

Reply:
That is perfect!